roles/ceph-validate/tasks/main.yml

```diff
@@ -52,7 +52,7 @@
   when:
     - osd_group_name in group_names
     - not osd_auto_discovery | default(False)
-    - osd_scenario != "lvm" and devices is not defined
+    - osd_scenario != "lvm" or (osd_scenario == "lvm" and devices is defined)
```

Comment:
I think it'd be safe to just say ``- devices is defined`` here.

Reply:
Ya, We can simplify as you have mentioned, just that when user chooses to use user created lvm_volumes, these tasks will be redundant
It should be fine

Reply:
In that case ``devices`` wouldn't be defined though, or shouldn't be. This works though.

Reply:
> I think it'd be safe to just say `- devices is defined` here.

Done - 4d17f8b573433d24b00e5d71b6d1ca2306cb416d